tests/lib/bin/session-tool

```diff
@@ -246,7 +246,7 @@ trap 'rm -rf $tmp_dir' EXIT
 # Run dbus-monitor waiting for systemd JobRemoved signal. Process the output
 # with awk, forwarding the result of the service to a fifo.
 monitor_expr="type='signal', sender='org.freedesktop.systemd1', interface='org.freedesktop.systemd1.Manager', path='/org/freedesktop/systemd1', member='JobRemoved'"
-stdbuf -oL dbus-monitor --system --monitor "$monitor_expr" > "$tmp_dir/dbus-monitor.pipe" &
+stdbuf -oL dbus-monitor --system --monitor "$monitor_expr" > "$tmp_dir/dbus-monitor.pipe" 2>/dev/null &
```

Comment:
Maybe we should collect the stderr to a file under `$tmp_dir` just in case?

Reply:
We could but what would happen with it next?

Reply:
As discussed on IRC, the directory is removed in all scenarios.

Reply:
I can do a follow up that logs it to `$tmp_dir`, for consistency. We remove that directory when `session-tool` exits so it's not a big improvement but it might be useful for local debugging.

Reply:
Done here https://github.com/snapcore/snapd/pull/8736